dispute.go

```diff
@@ -67,78 +67,78 @@ type DisputeListParams struct {
 	BillingAddress               string `json:"billing_address"`
 	CancellationPolicy           *File  `json:"cancellation_policy"`
 	CancellationPolicyDisclosure string `json:"cancellation_policy_disclosure"`
 	CancellationRebuttal         string `json:"cancellation_rebuttal"`
-	CustomerComm                 *File  `json:"customer_communication"`
-	CustomerEmail                string `json:"customer_email_address"`
-	CustomerIP                   string `json:"customer_purchase_ip"`
+	CustomerCommunication        *File  `json:"customer_communication"`
+	CustomerEmailAddress         string `json:"customer_email_address"`
 	CustomerName                 string `json:"customer_name"`
-	CustomerSig                  *File  `json:"customer_signature"`
-	DuplicateCharge              string `json:"duplicate_charge_id"`
-	DuplicateChargeDoc           *File  `json:"duplicate_charge_documentation"`
-	DuplicateChargeReason        string `json:"duplicate_charge_explanation"`
-	ProductDesc                  string `json:"product_description"`
+	CustomerPurchaseIP           string `json:"customer_purchase_ip"`
+	CustomerSignature            *File  `json:"customer_signature"`
+	DuplicateChargeDocumentation *File  `json:"duplicate_charge_documentation"`
+	DuplicateChargeExplanation   string `json:"duplicate_charge_explanation"`
+	DuplicateChargeID            string `json:"duplicate_charge_id"`
+	ProductDescription           string `json:"product_description"`
 	Receipt                      *File  `json:"receipt"`
 	RefundPolicy                 *File  `json:"refund_policy"`
 	RefundPolicyDisclosure       string `json:"refund_policy_disclosure"`
-	RefundRefusalReason          string `json:"refund_refusal_explanation"`
+	RefundRefusalExplanation     string `json:"refund_refusal_explanation"`
 	ServiceDate                  string `json:"service_date"`
-	ServiceDoc                   *File  `json:"service_documentation"`
+	ServiceDocumentation         *File  `json:"service_documentation"`
 	ShippingAddress              string `json:"shipping_address"`
 	ShippingCarrier              string `json:"shipping_carrier"`
 	ShippingDate                 string `json:"shipping_date"`
-	ShippingDoc                  *File  `json:"shipping_documentation"`
-	ShippingTracking             string `json:"shipping_tracking_number"`
+	ShippingDocumentation        *File  `json:"shipping_documentation"`
+	ShippingTrackingNumber       string `json:"shipping_tracking_number"`
 	UncategorizedFile            *File  `json:"uncategorized_file"`
 	UncategorizedText            string `json:"uncategorized_text"`
 }
 
 // File represents a link to downloadable content.
 type File struct {
-	Created int64  `json:"created"`
-	ID      string `json:"id"`
-	Mime    string `json:"mime_type"`
-	Purpose string `json:"purpose"`
-	Size    int    `json:"size"`
-	URL     string `json:"url"`
+	Created  int64  `json:"created"`
+	ID       string `json:"id"`
+	MimeType string `json:"mime_type"`
```

Comment:
Should we do `MIMEType` (**M**ultipurpose **I**nternet **M**ail **E**xtensions).

Reply:
fixed!